tf2onnx/rewriter/eye_rewriter.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+tf2onnx.rewriter.eye_rewriter - supports tf.eye
+"""
+
+from onnx import onnx_pb
+from tf2onnx.graph_matcher import OpTypePattern, GraphMatcher
+
+# pylint: disable=invalid-name,unused-argument,missing-docstring, unused-variable
+
+
+def rewrite_eye(g, ops):
+    # tf.eye is implemented by a sub_graph which contains op "MatrixDiag" or "MatrixSetDiag" while
+    # these two ops are un-supported directly in onnx
+    # but onnx op EyeLike can be used to map the sub_graph
+    # "rewrite_eye" supports tf.eye(tf.shape(x)[i]) and tf.eye(tf.shape(x)[i], tf.shape(x)[j]).
```

Comment:
tf.eye(tf.shape(x)[i]) => tf.eye(num_rows)
tf.eye(tf.shape(x)[0], tf.shape[1]) => tf.eye(num_rows, num_columns) ?

to align with tf.eye signature.
tf.eye(
    num_rows,
    num_columns=None,
    batch_shape=None,
    dtype=tf.dtypes.float32,
    name=None
)